src/litestorepkg/niftylogger.nim

```diff
@@ -0,0 +1,72 @@
+import 
```

Comment:
Why is niftylogger needed in this projects? Not keen of having it duplicated here. Currently the only semi-tidy way to include it adding it as a nifty package like I do in other projects, but LiteStore has been kept "nifty-free" for now...